src/main/java/com/dev/cinema/model/dto/TicketResponseDto.java

```diff
@@ -0,0 +1,9 @@
+package com.dev.cinema.model.dto;
+
+import lombok.Data;
+
+@Data
+public class TicketResponseDto {
+    private Long id;
+    private MovieSessionResponseDto movieSessionDto;
```

Comment:
let's not make a multilevel json objects. Take all information from movie session and make it TicketResponseDto `s fields